wpeverest/user-registration
# UR-1139 Refactor - Handle user status properly

### All Submissions:






### Changes proposed in this Pull Request:

This PR fixes some bugs related to the handling of user status. Previously, there were issues in the counting of pending users in notice and filtering of users based on user status.

Also, this PR saves the login option of the form in the user meta during registration.

### How to test the changes in this Pull Request:

1. Try to replicate the above-mentioned bugs before checking out to this branch.
2. After checking out to this branch, verify if the issues are fixed.
3. Verify if the user status filter on the Users page is working properly.
4. Try changing the login option of the form after registering some users. Make sure those users are handled according to the login option that was set during the time of their registration. ( Only for new users )
5. Make sure that no back compatibility issues arise for previously registered users.

### Types of changes:




### Other information:




### Changelog entry

> UR-1139 Refactor - Handle user status properly
